src/utils/__tests__/is-firefox.test.mjs

```
@@ -0,0 +1,94 @@
+    // Or, redefine it based on its original state if that's critical.
+    // Often, just restoring the spy is enough for isolated tests.
+    // If global.navigator.userAgent was originally a value property:
+    if (initialUserAgent !== undefined) {
+         Object.defineProperty(global.navigator, 'userAgent', {
+            value: initialUserAgent,
+            configurable: true,
+            writable: true, // Assuming it was writable
+        });
+    } else {
+        // If it didn't exist, or to be very clean, delete it
+        delete global.navigator.userAgent;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)